app/uk/gov/hmrc/helptosavestub/controllers/NSIController.scala

```diff
@@ -107,6 +112,75 @@ object NSIController extends BaseController with Logging {
 
   private val ninoStatusRegex = """AS(\d{3}).*""".r
 
+  def getAccount(correlationId: Option[String], nino: Option[String], version: Option[String], systemId: Option[String]): Action[AnyContent] = Action { implicit request ⇒
+    validateParams(correlationId, nino, version, systemId)
+      .fold(
+        errors ⇒ {
+          logger.warn("[Handle Account Query] invalid params")
+          errors match {
+            case error :: Nil ⇒ BadRequest(Json.toJson(error))
+            case _            ⇒ BadRequest
+          }
+        }, {
+          case nino ⇒
+            if (nino.contains("401")) {
+              Unauthorized
+            } else if (nino.contains("500")) {
+              InternalServerError
+            } else {
+              val maybeAccount = getAccountByNino(nino, correlationId)
+              maybeAccount match {
+                case Right(a) ⇒ Ok(Json.toJson(a))
+                case Left(e)  ⇒ BadRequest(Json.toJson(e))
+              }
+            }
+          case other ⇒
+            BadRequest
+        })
+  }
+
+  def getTransactions: Action[AnyContent] = ???
```

Comment:
probably best to just remove these endpoints if they're not implemented yet

Reply:
done